package.json

```diff
@@ -2,12 +2,16 @@
   "name": "pdf-generation-actions-web-component",
   "version": "1.11.0",
   "description": "components actions to print or render the report to a pdf",
-  "main": "dist/index.js",
+  "main": "index.js",
+  "module": "index.js",
```

Comment:
I don't think we need this flag, we only have one bundle.

Reply:
Modifed